<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradeCalc</class>
 <widget class="QMainWindow" name="GradeCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GradeCalc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="AssSlider1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="AssSlider2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="AssLabel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Assignment 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="AssLabel2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Assignment 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="AssLabel3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Assignment 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Course</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AssSpinBox1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="AssSpinBox2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="AssSlider3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="AssSpinBox3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="MidSpinBox1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="FinSpinBox1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="MidLabel1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinLabel1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Final</string>
    </property>
   </widget>
   <widget class="QSlider" name="MidSlider1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="FinSlider1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="SchemaRadioButton1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SchemaRadioButton2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema B</string>
    </property>
   </widget>
   <widget class="QLabel" name="OSLabel1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="OSLabel2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AssSpinBox6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="AssLabel4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Assignment 4</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AssSpinBox5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="AssSpinBox4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="AssLabel5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Assignment 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="AssLabel6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Assignment 6</string>
    </property>
   </widget>
   <widget class="QSlider" name="AssSlider6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="AssSlider4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="AssSlider5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="MidSlider2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="MidSpinBox2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="MidLabel2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>239</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Schemas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TestLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TestLabel</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>AssSlider1</zorder>
   <zorder>AssSlider2</zorder>
   <zorder>AssLabel1</zorder>
   <zorder>AssLabel2</zorder>
   <zorder>AssLabel3</zorder>
   <zorder>label_4</zorder>
   <zorder>AssSpinBox1</zorder>
   <zorder>AssSpinBox2</zorder>
   <zorder>AssSlider3</zorder>
   <zorder>AssSpinBox3</zorder>
   <zorder>MidSpinBox1</zorder>
   <zorder>FinSpinBox1</zorder>
   <zorder>MidLabel1</zorder>
   <zorder>FinLabel1</zorder>
   <zorder>MidSlider1</zorder>
   <zorder>FinSlider1</zorder>
   <zorder>SchemaRadioButton1</zorder>
   <zorder>SchemaRadioButton2</zorder>
   <zorder>OSLabel1</zorder>
   <zorder>OSLabel2</zorder>
   <zorder>AssSpinBox6</zorder>
   <zorder>AssLabel4</zorder>
   <zorder>AssSpinBox5</zorder>
   <zorder>AssSpinBox4</zorder>
   <zorder>AssLabel5</zorder>
   <zorder>AssLabel6</zorder>
   <zorder>AssSlider6</zorder>
   <zorder>AssSlider4</zorder>
   <zorder>AssSlider5</zorder>
   <zorder>MidSlider2</zorder>
   <zorder>MidSpinBox2</zorder>
   <zorder>MidLabel2</zorder>
   <zorder>TestLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
